test/test_wordcloud.py

```diff
@@ -41,19 +41,42 @@
     46 09 55 05 82   23 17 25 35 94   08 128
 """
 
+STOPWORDED_COLLOCATIONS = """
+thank you very much
+thank you very much
+thank you very much
+thanks
+"""
+
+SMALL_CANVAS = """
+better late than never someone will say
+"""
+
 
 def test_collocations():
-    wc = WordCloud(collocations=False, stopwords=[])
```

Comment:
Is this change required?

Reply:
Stopwords [is documented to be a set](https://github.com/amueller/word_cloud/blob/master/wordcloud/wordcloud.py#L204).  In Python `[]` is an empty array.  So I was updating it to use the correct type.